embedded-hal-async/CHANGELOG.md

```diff
@@ -7,8 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+No unreleased changes
+
+## [v1.0.0-rc.2] - 2023-11-28
+
+- Updated `embedded-hal` to version `1.0.0-rc.2`.
 - Minor document fixes.
 - Add #[inline] hints to most of `embedded-hal-async` functions.
+- delay: Rename `DelayUs` to `DelayNs`
+- delay: Add `DelayNs::delay_ns()`
+- delay: Add default impls of `delay_ms` and `delay_us` based on `delay_ns`.
+- spi: Rename `Operation::DelayUs` to `Operation::DelayNs`, with nanosecond precision.
+- spi: Rename `Operation::DelayUs` to `Operation::DelayNs`, with microsecond precision.
```

Comment:
?

Reply:
Good spot! I think this can occur because of https://github.com/rust-embedded/embedded-hal/blob/master/.gitattributes#L1, we see it in esp-hal sometimes.

Reply:
fixed